extensions/mssql/package.nls.json

```
@@ -175,5 +175,6 @@
 	"title.newTable": "New Table",
 	"title.designTable": "Design",
 	"mssql.parallelMessageProcessing" : "[Experimental] Whether the requests to the SQL Tools Service should be handled in parallel. This is introduced to discover the issues there might be when handling all requests in parallel. The default value is false. Relaunch of ADS is required when the value is changed.",
+	"mssql.enableSqlAuthenticationProvider" : "Enables registering Sql Authentication Provider implementation for 'Active Directory Interactive' authentication mode when user selects 'AzureMFA' authentication. This enables Server-side resource endpoint integration when fetching access tokens.",
```

I think this reads a little better. 

Is there some docs you can link to for this? I'm not sure most people are going to understand what this setting means so providing some way for them to find out more if they're curious would be nice. 

```suggestion
	"mssql.enableSqlAuthenticationProvider" : "Enables the Sql Authentication Provider for 'Active Directory Interactive' authentication mode when user selects 'AzureMFA' authentication. This enables Server-side resource endpoint integration when fetching access tokens.",
```

---

Not sure if this is something that should be explained further.. this is between driver and tools and how we implemented it, and this design is expected from our tools IMO. End users shouldn't have to investigate why they couldn't connect to certain endpoints as this was our design gap.

This configuration is only until we get confidence and stability in design and enable it by default - it will be removed in future.